srcpkgs/libaccounts-glib/template

```diff
@@ -1,21 +1,19 @@
 # Template file for 'libaccounts-glib'
 pkgname=libaccounts-glib
-version=1.23
+version=1.24
 revision=1
-build_style=gnu-configure
+build_style=meson
+wrksrc="${pkgname}-VERSION_${version}-8948717702424ce15f4e23e5db2c8ce0799ec120"
```

Comment:
wrksrc= directly below revision=

Reply:
Someone should really make a new release of xtools with the updated xlint...